autoload/OmniSharp.vim

```diff
@@ -390,6 +390,9 @@ function! OmniSharp#StartServer()
 			break
 		endif
 		let solutionfiles = globpath(folder , "*.sln", 1)
+        if isdirectory(solutionfiles)
```

Comment:
Sorry: I had configured my Vim to expandtab, and use tabstop=4 (while your code were with actual tab chars - to me it appeared as indented). I have corrected code indentation with noexpandtab.